frontend2/src/components/team/EligibilitySettings.tsx

```diff
@@ -76,7 +76,7 @@ const EligibilitySettings: React.FC = () => {
     !isPresent(team) ||
     !isPresent(episode)
   ) {
-    return "Error: You're not in a team!";
+    return <div></div>;
```

Comment:
We can place the string `Error: You're not in a team!` inside the div here to display it